.github/workflows/styling.yml

```diff
@@ -12,7 +12,7 @@ permissions:
 jobs:
 
   php_styling:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
```

Comment:
hi @kojiromike , nice. Maybe we should be running `php 8.4` as well? Maybe it'd be more efficient to not fire up 2 separate VMs for the other jobs, `css-styling` and `js-liniting`?

Reply:
@stephenwaite did you mean to make that comment in #8629?

Reply:
also we might want to try node 22 with ubuntu 24